goinstant/features/service_destroy.feature

```diff
@@ -0,0 +1,4 @@
+Feature: destroy service
+  Scenario: destroy the service                   
+    When the service is destroyed
+    Then check the service is destroyed
```

Comment:
```suggestion
    Then check the service is destroyed

```

Reply:
@Mark-Labuschagne Please go through this - https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log